README.md

```diff
@@ -149,15 +149,35 @@ Custom parser must be a singleton class that has a `.call` method that accepts t
 
 There are several strategies to get tag statistics
 
+### ActiveRecord Strategy (Default)
+
+Data about taggings is accessible via class methods of your model with `Metka::Model` attached. You can calculate a cloud for a single tagged column or multiple columns, the latter case would return to you a sum of taggings from multiple tagged columns, that are provided as arguments, for each tag present. ActiveRecord Strategy is an easiest way to implement, since it wouldn't require any additional code, but it's the slowest one on SELECT.
+
+```ruby
+class Book < ActiveRecord::Base
+  include Metka::Model(column: 'authors')
+  include Metka::Model(column: 'co_authors')
+end
+
+tag_cloud = Book.author_cloud
+=> [["L.N. Tolstoy", 3], ["F.M. Dostoevsky", 6]]
+genre_cloud = Song.co_author_cloud
+=> [["A.P. Chekhov", 5], ["N.V. Gogol", 8], ["L.N. Tolstoy", 2]]
+summary_cloud = Song.metka_cloud('authors', 'co_authors')
+=> [["L.N. Tolstoy", 5], ["F.M. Dostoevsky", 6], ["A.P. Chekhov", 5], ["N.V. Gogol", 8]]
+```
+
 ### View Strategy
```

Comment:
I'm also not fully sure, if we should leave View Strategy be or completely remove it

Reply:
@Bogda, close this comment?